go-controller/pkg/ovn/policy_test.go

```diff
@@ -2311,3 +2356,43 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Low-Level Operations", func() {
 		gomega.Expect(gp.delNamespaceAddressSet(four.GetObjectID(libovsdbops.ObjectNameKey))).To(gomega.BeFalse())
 	})
 })
+
+// buildNetworkPolicyPeerAddressSet builds the addresssets for the networkpolicy peer provided
+func buildNetworkPolicyPeerAddressSets(namespaceName string, peer knet.NetworkPolicyPeer, ips ...string) (*nbdb.AddressSet, *nbdb.AddressSet) {
```

Comment:
I see potential to use this also in `ginkgo.It("creates one address set for multiple users with the same selector", func() {` that test

Reply:
can probably also be used there but that would involve removing the FakeAddressSet from that test file next.

Reply:
oh that's another PR?

Reply:
that will be another PR, I am going one at a time to keep the changes manageable